### PR TITLE
Run TensorFlow related examples on Python3.8

### PR DESCRIPTION
**Pull request by crcrpar**

### CODE REVIEW

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ name: examples
 on:
   schedule:
     - cron: '0 15 * * *'
+  # TODO (crcrpar): Remove the below once CI succeeds.
+  pull_request: {}
```
Comment (crcrpar) [inline .github/workflows/examples.yml:7]: To me: Once CI passes, commit this.
```suggestion
```

**Comment by HideakiImamura** [comment]
@not522 @himkt Could you review this PR?

**Review by not522 — APPROVED** [review]
LGTM!

**Review by toshihikoyanase — APPROVED** [review]
The change looks good to me.
A nit-picky, but I guess we can merge the `elif` clause for Python 3.8 to the `else` clause. If you have not enough time, I'll send a follow-up PR.

**Comment by toshihikoyanase** [comment]
@himkt Let me merge this PR since it got two approvals. Thanks.

File: .github/workflows/examples.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ jobs:
         if [ ${{ matrix.python-version }} = 3.6 ]; then
           IGNORES='chainermn_.*|fastai*|pytorch_distributed_.*|rapids_.*|botorch_.*|pytorch/pytorch_checkpoint*'
         elif [ ${{ matrix.python-version }} = 3.8 ]; then
-          IGNORES='chainermn_.*|keras_.*|pytorch_distributed_.*|tensorboard_.*|tensorflow_.*|tfkeras_.*|fastai*|rapids_.*|pytorch/pytorch_checkpoint*'
+          IGNORES='chainermn_.*|pytorch_distributed_.*|fastai*|rapids_.*|pytorch/pytorch_checkpoint*'
```
[the remark this answers is not in the document]
Comment (toshihikoyanase) [reply .github/workflows/examples.yml:56]: I'll submit a follow-up PR for this.

**Pull request merged**